src/client/src/css/jobListing.css

```diff
@@ -173,7 +173,198 @@
   justify-content: space-between;
   align-items: center;
   margin-bottom: 20px;
+  padding: 20px;
+  background-color: #f0f0f0;
+}
+
+.jobFilter form {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  width: 100%;
+}
+
+.jobFilter label {
+  margin-right: 10px;
+  font-size: large;
+}
+
+@media all and (max-width: 1024px){
```

Comment:
For screen widths between 906 and 1024, this is the viewport I get.
![image](https://user-images.githubusercontent.com/78051064/231323957-532739a1-502c-42fd-b952-adca5e185fb1.png)

This is the viewport with a less tall screen
![image](https://user-images.githubusercontent.com/78051064/231324042-8c20fe5d-4bb0-4e1f-bde6-6e730c4645dd.png)